.vscode/launch.json

```diff
@@ -86,6 +86,44 @@
       },
       "preLaunchTask": "Compile"
     },
+    {
+      "name": "Launch Current Integration Test with sfdx-simple",
```

Comment:
Lets change the name to `Launch Current Core Integration Test with sfdx-simple` since these launch configs are specific to tests in the core module.

Reply:
Let's do the same thing for the lwc recipes launch config- `Launch Current LWC Integration Test with lwc-recipes`

Reply:
This worked for me running a test in the soql package.  Is it supposed to only work in core?

Reply:
@lcampos This one works for all but LWC extension. @AnanyaJha I can update the second one so it's more informative